motors/tacho_motor_class.c

```diff
@@ -255,7 +251,7 @@ struct tacho_motor_state_item {
 
 static struct tacho_motor_value_names tacho_motor_states[NUM_TM_STATES] = {
 	[TM_STATE_RUNNING]	= { "running" },
-	[TM_STATE_RAMPING]	= { "ramping" },
+	[TM_STATE_STOPPED]	= { "stopped" },
```

Comment:
The app knows that the stop command was sent in this particular example, but not if you are using run-timed. But, if you don't think it is useful, we can leave it out. 

Also, when you get into multi-threaded applications, you can't assume anything about what command was sent or what the stop command was.